package.json

```diff
@@ -24,66 +24,66 @@
-    "eslint-plugin-vuetify": "^2.5.2",
-    "glob": "^11.0.1",
-    "husky": "^9.1.7",
-    "inquirer": "^12.5.2",
-    "lerna": "^8.2.2",
-    "magic-string": "^0.30.17",
-    "mkdirp": "^3.0.1",
-    "moment": "^2.30.1",
-    "rimraf": "^6.0.1",
-    "sass": "^1.86.3",
-    "sass-embedded": "^1.86.3",
-    "semver": "^7.7.1",
-    "shelljs": "^0.9.2",
-    "stringify-object": "^5.0.0",
-    "typescript": "~5.5.4",
-    "upath": "^2.0.1",
-    "vite-plugin-inspect": "11.0.0",
-    "vite-plugin-warmup": "^0.1.0",
-    "vue": "^3.5.13",
-    "vue-tsc": "^2.0.29",
-    "yargs": "^17.7.2"
+    "@babel/cli": "catalog:",
+    "@babel/core": "catalog:",
+    "@babel/preset-env": "catalog:",
+    "@babel/preset-typescript": "catalog:",
+    "@mdi/font": "catalog:",
+    "@mdi/js": "catalog:",
+    "@mdi/svg": "catalog:",
+    "@octokit/core": "catalog:",
+    "@types/yargs": "catalog:",
+    "@typescript-eslint/eslint-plugin": "catalog:",
+    "@typescript-eslint/parser": "catalog:",
+    "@unhead/vue": "catalog:",
+    "@vue/compiler-sfc": "catalog:",
+    "@vue/language-server": "catalog:",
+    "@vue/runtime-core": "catalog:",
+    "@vue/runtime-dom": "catalog:",
+    "@vueuse/head": "catalog:",
+    "babel-eslint": "catalog:",
+    "conventional-changelog-cli": "catalog:",
+    "conventional-changelog-vuetify": "catalog:",
+    "conventional-github-releaser": "catalog:",
+    "cross-spawn": "catalog:",
+    "eslint": "catalog:",
+    "eslint-config-standard": "catalog:",
+    "eslint-formatter-codeframe": "catalog:",
+    "eslint-plugin-import": "catalog:",
+    "eslint-plugin-local-rules": "catalog:",
+    "eslint-plugin-n": "catalog:",
+    "eslint-plugin-promise": "catalog:",
+    "eslint-plugin-react": "catalog:",
+    "eslint-plugin-sonarjs": "catalog:",
+    "eslint-plugin-vue": "catalog:",
+    "eslint-plugin-vuetify": "catalog:",
+    "glob": "catalog:",
+    "husky": "catalog:",
+    "inquirer": "catalog:",
+    "lerna": "catalog:",
+    "magic-string": "catalog:",
+    "mkdirp": "catalog:",
+    "moment": "catalog:",
+    "rimraf": "catalog:",
+    "sass": "catalog:",
+    "sass-embedded": "catalog:",
+    "semver": "catalog:",
+    "shelljs": "catalog:",
+    "stringify-object": "catalog:",
+    "typescript": "catalog:",
+    "upath": "catalog:",
+    "vite-plugin-inspect": "catalog:",
+    "vite-plugin-warmup": "catalog:",
+    "vue": "catalog:",
+    "vue-tsc": "catalog:",
+    "yargs": "catalog:"
   },
   "packageManager": "pnpm@10.8.0",
   "pnpm": {
     "patchedDependencies": {
-      "@mdi/js@7.4.47": "patches/@mdi__js@7.4.47.patch",
-      "@testing-library/vue": "patches/@testing-library__vue.patch",
-      "vue-gtag-next": "patches/vue-gtag-next.patch"
+      "@mdi/js@7.4.47": "/home/john/sites/vuetify/patches/@mdi__js@7.4.47.patch",
+      "@testing-library/vue": "/home/john/sites/vuetify/patches/@testing-library__vue.patch",
+      "vue-gtag-next": "/home/john/sites/vuetify/patches/vue-gtag-next.patch"
```

Comment:
Remove prefix `/home/john/sites/vuetify/`, should be relative.

We also need to update to pnpm 10.8.1, `pnpm patch` fixed in this release: https://github.com/pnpm/pnpm/releases